tests/test_motion.py

```diff
@@ -17,7 +18,9 @@
 VELOCITY_SET_POINT_REGISTER = "CL_VEL_SET_POINT_VALUE"
 ACTUAL_VELOCITY_REGISTER = "CL_VEL_FBK_VALUE"
 CURRENT_QUADRATURE_SET_POINT_REGISTER = "CL_CUR_Q_SET_POINT"
+ACTUAL_QUADRATURE_CURRENT_REGISTER = "CL_CUR_Q_VALUE"
```

Comment:
Maybe we could create a constants file where we could define these values and import them where needed, instead of creating the same variables on two different files.